tox.ini

```diff
@@ -83,8 +104,8 @@ commands =
     all  : python -m sphinx {[docs]sphinx_args_extra} -b html . {[docs]build_dir}/html
 
 [testenv:package]
-package_env = none
```

Comment:
This [workaround](https://github.com/beeware/briefcase/pull/995/files#r1042849461) doesn't seem necessary [anymore](https://github.com/tox-dev/tox/pull/2800).